atlp-rwanda/atlp-devpulse-fn
# ATLP Devpulse sidebar

## What does this PR do?
Designing the ATLP Devpulse sidebar

## Description of Task to be completed?

Devpulse sidebar

## How should this be manually tested?

- `Clone the repo`

- `checkout` to branch `ft-sidebar`

- Pull the changes by `git pull`

- Install all packages by running `npm install`

- Start app by `running npm run dev`

-navigate to localhost:http://localhost:3000/sidebar route to see the sidebar.

## Any background context you want to provide?

disregard the background difference between figma design and the actual design it was used to show sidebar borders and limits,it will be altered during defining dashboard layout

## What are the relevant pivotal tracker stories?

[Trello Board reference](https://trello.com/c/Rr46fTht/54-dashboard-sidebar)
